app-android/src/main/java/com/ustadmobile/port/android/view/ContentEntryImportLinkFragment.kt

```diff
@@ -83,4 +103,67 @@ class ContentEntryImportLinkFragment : UstadBaseFragment(), ContentEntryImportLi
     }
 
 
+}
+
+@Composable
+fun ContentEntryImportLinkScreen(
+    uiState: ContentEntryImportLinkUiState,
+    onClickNext: () -> Unit = {},
+    onUrlChange: (Url?) -> Unit = {}
```

Comment:
This needs to use a string type. The user might type an invalid URL (the final URL might be invalid, or it might be something is invalid as typed and then the user changes it). 

This code would crash in that case. For example let's say the user starts typing a link, and there is just one letter. One letter on its own is not a valid URL. The onValueChange function will run, calling Url("h") will crash.